charmcraft/templates/init-kubernetes/tests/unit/test_charm.py.j2

```diff
@@ -3,22 +3,19 @@
 #
 # Learn more about testing at: https://juju.is/docs/sdk/testing
 
-import ops
-import ops.testing
-import pytest
+from ops import testing
+
 from charm import {{ class_name }}
 
 
-@pytest.fixture
-def harness():
-    harness = ops.testing.Harness({{ class_name }})
-    harness.begin()
-    yield harness
-    harness.cleanup()
+def test_pebble_ready():
+    # Arrange:
```

Comment:
**TIL** https://mitesh1612.github.io/blog/2020/11/27/different-types-UTs

I see that it's used in Scenario and documented at https://juju.is/docs/sdk/write-a-scenario-test-for-a-charm-library